index.html

```diff
@@ -35,9 +36,10 @@
       <header class="header">
         <a href="https://embark-studios.com"><img id="logo" src="./img/logo.png"/></a>
         <div>
-          <a href="https://embark-studios.com" class="fa fa-globe"></a>
+          <a href="https://embark-studios.com" class="fas fa-globe-americas"></a>
           <a href="https://twitter.com/EmbarkStudios" class="fa fa-twitter"></a>
           <a href="https://github.com/EmbarkStudios" class="fa fa-github"></a>
+          <a href="https://discord.gg/ma4NTtB" class="fab fa-discord"></a>
```

Comment:
Let's use this link, since it's a permanent one that I know won't be deleted/revoked: `https://discord.gg/8TW9nfF`